src/Windows/R/Components/Impl/ConnectionManager/Implementation/ViewModel/ConnectionManagerViewModel.cs

```diff
@@ -326,13 +326,12 @@ internal sealed class ConnectionManagerViewModel : ConnectionStatusBaseViewModel
                 _localDockerConnections.Clear();
                 _remoteConnections.Clear();
                 foreach (var connection in ConnectionManager.RecentConnections.OrderBy(c => c.Name)) {
-                    var cvm = CreateConnectionViewModel(connection);
                     if (connection.IsRemote) {
```

Comment:
Prob will benefit from some class derivation and virtual factories, but OK for now

Reply:
Not just factories, but actually types that handle different types of operations. #4029